stdlib/2and3/socket.pyi

```diff
@@ -559,6 +560,14 @@ class socket:
     def setsockopt(self, level: int, optname: int, value: Union[int, bytes]) -> None: ...
     def shutdown(self, how: int) -> None: ...
 
+    if sys.version_info >= (3, 3):
+        def recvmsg(self, __bufsize: int, __ancbufsize: Optional[int] = ...,
```

Comment:
I don't think these are Optional (`sock.recvmsg(1, None)` fails with a TypeError).

Reply:
You are totally right. For some reason I got it in my head that all default arguments needed to be Optional as well. Something about those ellipsis and the statement in the contributing guide regarding being implicit about Optionals. Pretty sure I did the same nonsense in the os module stubs as well. Thanks for catching this.